main.go

```diff
@@ -56,4 +61,6 @@ func Init(context map[string]interface{}) {
 			panic(err)
 		}
 	}()
+
+	return GorillazContext{router: router}
```

Comment:
The idea is good, but we already started the HTTP server in another goroutine.
I have no idea what will happen if we try to register another route once the http server already runs.

Reply:
My guess is that we should split gorillaz.Init in 2:
+ gaz.NewGaz()
(which would return a gaz, with a router)
and
+ gaz.Run() which would run the http.ListenAndServe()

Or we should remove the http.ListenAndServe and let the gorillaz user do it

Reply:
good idea, I pushed another version